python/MRzeroCore/phantom/sim_data.py

```diff
@@ -66,7 +66,8 @@ def __init__(
         dephasing_func: Callable[[torch.Tensor, torch.Tensor], torch.Tensor],
         recover_func: Callable[[SimData], Any] | None = None,
         phantom_motion=None,
-        voxel_motion=None
+        voxel_motion=None,
+        Masks: Optional[Dict[str,torch.Tensor]] = None,

```

Comment:
Please change to lower-case. All variables except physical properties (T1, T2 etc)  are ``snake_case``

Reply:
the lower case `masks` was a bit confusing since `voxel_grid_phantom` also uses `mask` for thresholding on `PD` so i changed the `Masks` into `tissue_masks`

Reply:
that's even better